src/components/Book.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import '../styles/Book.css';
+
+function Book() {
+  return (
+    <div className="book-container">
+      <h3>Genre</h3>
+      <h2>Book name</h2>
+      <p>Author name</p>
```

Comment:
- [ ] The `Book` component should take the book data as a prop and render the `title` and the `author`. Although you don't have data now, you can add some mock data as props in the `Books` component and call them here as variables instead of hardcoded values. Kindly fix this